Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM rust:1.40 as builder
+FROM rust:1.45.2 as builder
```

Comment:
why no use latest version released yesterday? Or is it substrate issue afaik they has some issue which required 2020-10-05 version. But I think I would wait for latest version of compiler ?

Reply:
I was going to update it, however clippy is panicing, see my comment here: https://github.com/AdExNetwork/adex-validator-stack-rust/issues/330#issuecomment-730882481

We don't use substrate in this repo, the issue arouse from compiler regression, this is why we reverted to 1.45.2 in the first place, I just forgot to update the `Dockerfile`

Reply:
See #349 for the `1.48.0` upgrade